app/views/pages/job-application-preview.html.slim

```diff
@@ -0,0 +1,211 @@
+- content_for :page_title_prefix do
```

Comment:
can we do it on one line instead like `- content_for :page_title_prefix, "Job Application Preview"`